doc/contributing/documentation_guide.md

```diff
@@ -216,14 +216,12 @@ may not render them properly.
 In particular, avoid building tables with HTML tags
 (<tt><table></tt>, etc.).
 
-Alternatives are:
-
-- The GFM (GitHub Flavored Markdown) table extension,
-  which is enabled by default. See
-  {GFM tables extension}[https://github.github.com/gfm/#tables-extension-].
```

Comment:
RDoc doe support GFM tables, but only in markdown files. For example, [doc/contributing/glossary.md](https://github.com/ruby/ruby/blob/master/doc/contributing/glossary.md) is generated at: https://docs.ruby-lang.org/en/master/contributing/glossary_md.html

Reply:
Thanks.  Can the doc for a single class/module/method/etc. be markdown?  Or must the entire file be markdown?  (Asking for a friend.)

Reply:
According to [the readme](https://github.com/ruby/rdoc/blob/master/README.rdoc#writing-documentation-):

> You can also set the comment format for a single file through the `:markup:` directive, but this is only recommended if you wish to switch markup formats.

So I think you can switch to markdown by using `:markup: markdown`.

Reply:
Modified.